src/float/add.rs

```diff
@@ -187,111 +187,28 @@ add!(__adddf3: f64);
 #[cfg(test)]
 mod tests {
     use core::{f32, f64};
+    use qc::{F32, F64};
 
-    use float::{Float, FRepr};
-    use qc::{U32, U64};
-
-    // TODO: Add F32/F64 to qc so that they print the right values (at the very least)
     check! {
         fn __addsf3(f: extern fn(f32, f32) -> f32,
-                    a: U32,
-                    b: U32)
-                    -> Option<FRepr<f32> > {
-            let (a, b) = (f32::from_repr(a.0), f32::from_repr(b.0));
-            Some(FRepr(f(a, b)))
+                    a: F32,
+                    b: F32)
+                    -> Option<F32> {
+            if option_env!("TARGET") != Some("arm-unknown-linux-gnueabi") {
```

Comment:
This doesn't seem to be working; Travis is still failing this test.

Could you instead `#[cfg]` away the `addsf3` and `adddf3` symbols along with these too tests for the `arm-unknown-linux-gnueabi`? And please add a comment about the problem and point to the issue that's tracking it #90